components/push/android/src/main/java/mozilla/appservices/push/LibPushFFI.kt

```diff
@@ -42,43 +40,43 @@ internal interface LibPushFFI : Library {
     // free them).
 
     /** Create a new push connection */
-    fun push_connection_new(
```

Comment:
The functions in this file can't be renamed, they must be the same as the names in the ffi/src/lib.rs file. You should instead suppress the warning. When I said push should use camelCase I meant for the argument names to PushManager's constructor, for example.

Reply:
I was in the process of cleaning those up. Calling the methods effectively `PushManager(...).push_connection_new()` vs. 'PushManager().newConnection()` for instance seemed a bit redundant and not super friendly to Kotlin devs. As I understand, the call pivots on the ffi lib.rs extern "C" call function names, which I'm guessing should map closer to the caller rather than the source style formatting.

Doing local builds and testing, the calls align and the tests pass. The two failures are due to packages outside of push (If I understand correctly, sync15 and fxa-client failed to build. I do not have permissions to rerun those tests.)

Reply:
Completely disagree. Consumers never should be calling these methods directly, they're entirely internal, so it doesn't matter what we call them. Changing it here requires us change our rust code to a nonstandard style, and we have far more rust code than Kotlin in our repo.

If one of them has to be nonstandard (and in this case one of them does), it should be the Kotlin.

Reply:
Agree completely, the rust side should be snake_case